Makefile

```diff
@@ -26,22 +26,24 @@ ISORT_OPTIONS := --profile black
 .PHONY: fmt
 fmt:
 	@echo "Running black fmt..."
-	python -m black $(BLACK_OPTIONS) $(CHECKED_IN_PYTHING_FILES)
-	python -m isort $(ISORT_OPTIONS) $(CHECKED_IN_PYTHING_FILES)
+	@python -m black $(BLACK_OPTIONS) $(CHECKED_IN_PYTHON_FILES)
```

Comment:
you are commenting out all of "make fmt"? what am i missing?

Reply:
It's not commenting it out, it's preventing the command itself from being printed.